summarization/text_media_matching/text_media_matching_helper.py

```diff
@@ -59,6 +59,7 @@ def get_text_media_matching(self):
             self.set_size)
 
         matchings = stable_matcher.get_matching()
+        matchings.sort()
```

Comment:
I don't see any changes related to sorting matchings here. Why do we need it?

Reply:
This is mainly to help testing since the expected matching are in sorted order of the sentences supplied to them!